packages/graphql/src/schema/resolvers/composition/wrap-query-and-mutation.ts

```diff
@@ -123,11 +123,15 @@ export const wrapQueryAndMutation =
             features,
             subscriptionsEnabled,
             executor,
-            neo4jDatabaseInfo,
             authorization: authorizationContext,
-            // Consider anything in here overrides
+        };
+
+        const finalContext = {
```

Comment:
Does this change reduce the flakiness of e2e tests?  If not I would remove this from this PR as there could be some side effects produced by this change unrelated to our tests that are worth seeing in isolation!